app/views/posts/index.html.erb

```diff
@@ -1,12 +1,20 @@
 <%= render 'shared/user', user: @user%>
-<ul>
+<ul class='all-posts'>
   <% if @posts.empty?%>
     <p class='post-card-no-click'>No posts</p>
   <% else%>
-  <div class='users-bio'><%=@user.bio%></div>
   <% @posts.each do |post|%>
     <li>
-      <%= render 'shared/post', post: post%>
+      <div class='posts-card-div'><%= render 'shared/post', post: post%></div>
+      <div class='posts-comments'>
+        <h3>Comments:</h3>
+        <% post.recent_5_comments.each do |comment,index| %>
+        <div>
+          <span class='comment-author'><%= comment.author.name %>:</span>
+          <span><%= comment.text %></span>
```

Comment:
- [ ] Great job so far @lorenzoworx However, when I click on `See all posts` on the `User show page` so I get navigated to the `posts index page` to check whether you have implemented the previous reviewers' changes, your app throws an error I have attached a screenshot below. I noticed that this was because in your table, your `Comments` are created with a method `content` and not `text` as you currently have here. So changing this from `text` to `content` should fix the error :wink: 
- Error Message
![image](https://github.com/lorenzoworx/velvetScribe/assets/102699958/f756d959-c6f6-432e-8fa6-6b53c23d2bf6)


- [ OPTIONAL] When trying to run your `seed file`, you made use of `text` for creating the comments there as well so it threw an error when I tried to run the seed file. So kindly fix that as well.
- [OPTIONAL ] Kindly make the same change in your `_comment.html.erb` file as well :wink: In this line below, kindly change `comment.text` to `comment.content`.

```ruby
  <li><span class='comment-author'><%=comment.author.name%></span>: <%=comment.text%></li>
```